install/kubernetes/citadel_extras/istio-citadel-plugin-certs.yaml.tmpl

```diff
@@ -7,13 +7,18 @@ metadata:
   name: istio-citadel-service-account
   namespace: {ISTIO_NAMESPACE}
 ---
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: istio-citadel
   namespace: {ISTIO_NAMESPACE}
+  labels:
+    istio: citadel
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+        istio: citadel
```

Comment:
Be careful of the indention here, although it's still working.

Reply:
Let me modify it